src/client/pages/Offer/Checkout/index.tsx

```diff
@@ -185,6 +185,14 @@ const isManualReviewRequired = (errors: GraphQLError[]) => {
   return manualReviewRequiredError !== undefined
 }
 
+const isSsnInvalid = (errors: GraphQLError[]) => {
+  const invalidSsnError = errors.find((error) => {
+    return error?.extensions?.body?.errorMessage === 'Invalid SSN'
```

Comment:
Hmmm this is never shown to users, right?

If it's more like an error code I would store it as a const first and reference that. Don't like hard coded strings 😅

Reply:
Yes, it's never shown to users. I am planning to get in touch with the backend teams as this should be rather a code instead. We are relying on an error message because the errorCode is empty.